app/jobs/hyku_knapsack/application_job.rb

```diff
@@ -2,5 +2,66 @@
 
 module HykuKnapsack
   class ApplicationJob < ActiveJob::Base
+    PRIORITY_QUEUE_NAME = :auxiliary
```

Comment:
These configurations are I believe generally useful for Hyku Prime; I suspect a better approach will be to extract a class variable that has hash table with key constant name (.e.g `"Bulkrax::ScheduledRelationshipsJob"`) and value of the integer.  This way downstream knapsacks can easily configure.

Why?  My understanding is that @laritakr and others have found these priorities to be the best performing configuration.